packages/building-utils/index-html/extract-resources.js

```diff
@@ -34,6 +34,8 @@ function extractResources(htmlString) {
     if (src) {
       jsModules.push(src);
       remove(moduleScript);
+    } else {
+      inlineModules.push(moduleScript);
```

Comment:
shouldn't inline module also be removed?

Reply:
Yes, I'll add it